docs/standard/class-libraries-multi-targeting.md

```diff
@@ -0,0 +1,283 @@
+---
+title: .NET Class Library Multi-Targeting
+description: Learn how to create .NET class libraries for multiple .NET platforms that can access and wrap platform-specific functionality.
+keywords: .NET, .NET Core
+author: immol
+ms.author: mairaw
+ms.date: 03/19/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+ms.assetid: 375da522-3520-4735-b1de-68c7b71719d5
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+
+# Introduction
+
+Even though .NET Standard 2.0 is very rich in terms APIs, there are rare cases
+where you need to access framework specific APIs (for instance, you might want
+to pop UI for starting an OAuth flow). There are also cases where you want to
+introduce a standard way to access framework-specific functionality (for
+example, using the geolocation APIs).
+
+You can do this via [multi-targeting](frameworks.md) which builds your project
+for multiple framework from a single project. This enables you to use
+conditional compilation with `#if` to call framework specific APIs.
+
+Please note that this topic is primarily about libraries. For application
+development we generally recommend to use multiple projects (one per
+application) and sharing code via Shared Projects.
+
+# Multi-targeting
+
+.NET Standard projects start as targeting a single framework, namely a specific
+version of .NET Standard:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+</Project>
+```
+
+In order to target multiple frameworks, you need to change the name of the
+property from `TargetFramework` to the plural form `TargetFrameworks`. Then you
```

Comment:
May want to list that you need to have these platforms installed on your machine for it to work... so if you add XamarinAndroid80 you need Android 8.0 sdk and xamarin installed.

Reply:
Would be nice if there was a way to express this in the targets/props in some way to make VS install it on demand.

Reply:
That's a good point, actually. I think we already have a similar experience in VS for acquiring the Windows SDK, for instance. Shouldn't be entirely hard to extend this.

@jamesmontemagno, would you mind filing this request in https://github.com/dotnet/project-system? Since you're the PM driver for Caboodle this seems topical. Happy to chime in and express my support and help with designing it.